lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -81,8 +77,9 @@ public EvaluateGpg45ScoresHandler() {
         this.configurationService = new ConfigurationService();
         this.userIdentityService = new UserIdentityService(configurationService);
         this.ipvSessionService = new IpvSessionService(configurationService);
-        this.gpg45ProfileEvaluator = new Gpg45ProfileEvaluator();
-        this.ciStorageService = new CiStorageService(configurationService);
+        this.gpg45ProfileEvaluator =
+                new Gpg45ProfileEvaluator(
+                        new CiStorageService(configurationService), configurationService);
```

Comment:
Something about this draws my attention. Should CiStorageService be initialised inside the Gpg45ProfileEvaluator if it's only being used there? Instead of within the Handler?

Reply:
I know what you mean about drawing your attention. But I don't want to closely couple the `CiStorageService` class to the `Gpgp45ProfileEvaluator` by having it initialised inside. By passing an instance of it in it keeps them loosley(sp?) coupled and easier to test.

Reply:
Makes sense Chris! Happy to keep it as it is 👍